Library/Homebrew/blacklist.rb

```diff
@@ -83,5 +83,8 @@ def blacklisted?(name)
     A binary installer is available:
       https://www.haskell.org/platform/mac.html
     EOS
+  when "mysqldump-secure" then <<-EOS.undent
+    The creator of mysqldump-secure tried to game our popularity metrics.
```

Comment:
Hi @mikemcquaid,
the code comment was no admission.
After reading the blacklist and the reasons why other projects got blacklisted, I found it a bit harsh to *completely* blacklist the project itself.

Maybe there is a way more *fair* way to simply blacklist the submission by me, so someone else could resubmitted it in the future, once it gains more popularity (If this is technically and easily possible).


I am already using a Tap now (cytopia/tap) for distributing.